### ISSUE TITLE
Feature request: Change room name, topic & image (avatar)

### ISSUE THREAD

**Issue by fkuersch** [Skylar-Tech/node-red-contrib-matrix-chat#97]
I'd like to change a room's name, topic & image (avatar) using node-red-contrib-matrix-chat. Just like the following commands in Element:

```
/roomname <name>
/topic <topic>
/roomavatar <mxc_url>
```

**Comment by skylord123** [comment]
Yeah this would be a nice addition. I think a node that allows you to change the various settings by sending a `msg.payload` containing an object with what you want change would be a great way to accomplish this.

Example:
```
msg.payload = {
	'name': 'New Room Name',
	'topic': 'New Room Topic',
	'avatar': '<mxc_url>',
	'publish_addresses': ['#my-room', '#another-room-address'],
	'url_previews': true
};
```

And if any of them are null/undefined we just ignore changing that setting.

**Comment by fkuersch** [comment]
Wow, you already implemented that feature. That's great news. I appreciate that. Thank you :)
